torch/csrc/jit/script/module.h

```diff
@@ -108,6 +109,7 @@ struct Method {
     for (auto inp : member_inputs) {
       inputs.push_back(*inp);
     }
+    EraseNumberTypes(retval);
```

Comment:
Ok, I don't like how in how many places this is appearing. This also looks like it is because PropagateInputShapes does not work with number types. I think the right fix is to add handling to PropagateInputShapes for the operators that you added. This can be done in a follow up PR, since we are blocked on other things waiting for number type.

Reply:
Will add to the list of followup prs

Reply:
Oh, this one here in particular is to patch ONNX export, I'll try to see if there's a better place to put it.